frontend/src/components/Header/Header.js

```diff
@@ -0,0 +1,31 @@
+import React, { Component } from 'react';
+import { NavLink } from 'react-router-dom';
+import './Header.css';
+
+
+class Header extends Component {
+  render() {
+    return (
+      <header>
+        <nav>
+          <ul className="nav-links">
+            <li className="nav-item">
+              <NavLink className="nav-link" to="/about" activeStyle={{background:'hsl(101,43%,42%)',color:'white'}}>About</NavLink>
```

Comment:
Is there a specific reason for the inline style? If not, let's move this to a CSS file.

Reply:
I replaced the inline activeStyle bits with activeClassName, and moved the styling declarations in the separate css file.